tests/acceptance/features/bootstrap/SharingNgContext.php

```diff
@@ -714,6 +714,26 @@ public function userEnablesSyncOfShareUsingTheGraphApi(string $user, string $sha
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When user :user tries to enable share sync of non-existing resource using the Graph API
+	 *
+	 * @param string $user
+	 *
+	 * @return void
+	 */
+	public function userTriesToEnablsShareSyncOfNonExistingItemUsingTheGraphApi(string $user):void {
```

Comment:
please, update similar changes to other linked lines as well
```suggestion
	public function userTriesToEnablsShareSyncOfNonExistingResourceUsingTheGraphApi(string $user):void {
```